.github/workflows/update_test_file_ratings.yml

```diff
@@ -2,8 +2,15 @@ name: Update test file ratings for TD experiment
 
 on:
   pull_request:
+    paths:
+      - ".github/workflows/update_test_file_ratings.yml"
+      - ".github/scripts/calculate_file_test_rating.py"
   schedule:
-    - cron: 5 2 * * 5  # At 02:05 on Friday UTC or about 7pm PT Thursday
+    - cron: 5 2 * * *  # At 02:05 UTC every day or about 7pm PT
```

Comment:
Consider moving to be closer to early in the am (like 4am PT) to offer slightly fresher data for people coming in the next morning